client/src/components/buttons/DeleteButton.js

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from 'react';
+import styled from '@emotion/styled';
+import colors from '../../utils/colors';
+import Delete from '../../assets/deleteButton.svg';
+import { CoinContext } from '../../utils/CoinContext';
+import PropTypes from 'prop-types';
+
+const DeleteImage = styled.img`
+  width: 100%;
+  height: 100%;
+`;
+
+const Button = styled.button`
+  border: none;
+  background-color: ${colors.backgroundPrimary};
+  outline: none;
+  padding: 0 4px 0 7px;
+  position: relative;
+  bottom: -23px;
+  &:active {
+    transform: scale(0.98);
+  }
+`;
+
+const ImageContainer = styled.div`
+  width: 25px;
+`;
+
+export default function DeleteButton({ name }) {
```

Comment:
This is a very specific button because it deletes a coin on click.
In this case, I would name it more specific too.

First, I didn't know what to expect when I saw this line:
`<DeleteButton name={title} />`
I thought `name` is the content of the button. 
Thus, it makes sense to rethink the naming.

Maybe like this?
`<DeleteCoinButton coinName={coin.name} />`